components/support/nimbus-fml/src/fixtures/intermediate_representation.rs

```diff
@@ -95,14 +95,38 @@ mod dump_to_file {
         Ok(())
     }
 
+    fn write_yaml(fm: &FeatureManifest, nm: &str) -> Result<()> {
+        let root = std::env::var("CARGO_MANIFEST_DIR")
+            .expect("Missing $CARGO_MANIFEST_DIR, cannot write fixtures files");
+        let fixtures_dir = "fixtures";
+        let path: PathBuf = [&root, fixtures_dir, nm].iter().collect();
```

Comment:
super nit: we can isolate a `get_fixture_path` function and use it in both `write_*` functions (or use generics, but with only two use cases, defs not worth it)